corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -279,6 +286,7 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
             singleApp: self.singleApp,
             sortIndex: self.sortIndex,
             forceLoginAs: self.forceLoginAs,
+            selectedValues: self.selectedValues,
```

Comment:
Since this is being added to `toJson`, it should also be added to `fromJson` and to the `CloudcareUrl` constructor.

Reply:
Nevermind - I don't think `selectedValues` should be part of the URL, will leave a more detailed comment in a minute.